pkgs/top-level/all-packages.nix

```diff
@@ -6680,6 +6682,8 @@ in
 
   Sylk = callPackage ../applications/networking/Sylk {};
 
+  otter-browser = qt5.callPackage ../applications/networking/browsers/otter {};
```

Comment:
otter-browser?

Reply:
I have no idea where that came from? I'll remove that.

Reply:
It has been axed.